lib/demo/accounts.ex

```diff
@@ -272,8 +272,8 @@ defmodule Demo.Accounts do
   def confirm_user(token) do
```

Comment:
Since this now returns `{:ok, user} | :error` should this function have examples in its docs?

Reply:
Your call. :)